CHANGELOG.md

```diff
@@ -10,9 +17,6 @@
 
 ### Bug Fixes
 
-* Avoid duplication in `enumerations` for same elements with different description
```

Comment:
This changelog entry was incorrectly added into a prior release. As you can see in the tagged changelog for the pre.8 release, this bug fix was not present: https://github.com/apiaryio/drafter/blob/20c2bdede10e8713db0036d58b422812192d35bc/CHANGELOG.md